steps/cli/oc.go

```diff
@@ -0,0 +1,22 @@
+package cli
+
+import (
+	"github.com/getgauge-contrib/gauge-go/gauge"
+	m "github.com/getgauge-contrib/gauge-go/models"
+	"github.com/openshift-pipelines/release-tests/pkg/oc"
+	"github.com/openshift-pipelines/release-tests/pkg/store"
+)
+
+var _ = gauge.Step("Create <table>", func(table *m.Table) {
```

Comment:
Isn't it better if this take resource location as parameter and the gauge takes care of parameterization? For example if I want to create only one resource for my test case and with the above step implementation we have to write a table for the single resource as well

Reply:
As per my use case I do perform same action with 2 different resources version `v1alpha1` & `v1beta1`  so I choose data driven approach here to fasten my execution, and also for creating multiple resources, we no need to duplicate steps

Reply:
I agree, my point is can we have a step like `create resource "resourceLocation"` and create a data table to create multiple resources? By this way the step can be used to create single resource as well.

Reply:
Like `Dynamic Parameters` in https://docs.gauge.org/writing-specifications.html?os=linux&language=python&ide=vscode

Reply:
yes, gauge has concept `Step alias` is a feature when there are multiple step names for the same step functionality.
eg: https://docs.gauge.org/writing-specifications.html?os=linux&language=java&ide=null